src/main/java/dev/emi/trinkets/TrinketsNetwork.java

```diff
@@ -4,6 +4,8 @@
 
 public class TrinketsNetwork {
 
-  public static final Identifier SYNC_SLOTS = new Identifier(TrinketsMain.MOD_ID, "sync_slots");
+  private TrinketsNetwork() {
```

Comment:
I tend to prefer fields above methods, including constructors, is this a different idiom for private constructors?

Reply:
I generated it with my IDE and for some reason it put it there by default.